types/recurly__recurly-js/index.d.ts

```diff
@@ -0,0 +1,555 @@
+// Type definitions for non-npm package recurly__recurly-js-browser 4.12
```

Comment:
If this is a non-npm package, why does it need to be scoped?

Reply:
Great question!

I wasn't sure how to do this honestly. A node package called [recurly-js](https://www.npmjs.com/package/recurly-js) exists, but it's unrelated to this one. Because this browser library lives inside of the recurly org on github, this felt like the best way to publish, but I would have no problem changing this to whatever would be most appropriate.